test/cypress/integration/29-host.dashboard.js

```diff
@@ -8,4 +8,81 @@ describe('host dashboard', () => {
     cy.get('.Orders .item:first .status').contains('paid');
     cy.wait(1000);
   });
+
+  it('approve expense and reject expense', () => {
+    cy.login({ redirect: '/brusselstogetherasbl/dashboard/expenses' });
+    // navigate to first item
+    cy.get('[data-cy="expense-paid"]').as('currentExpense');
+    cy.get('[data-cy="expense-actions"]')
+      .as('currentExpenseActions')
+      // click 'unapprove'
+      .contains('button', 'Unapprove')
+      .click({ force: true });
+    // modal pops up - click 'yes'
+    cy.get('[data-cy="confirmation-modal-continue"]').click({ force: true });
+    cy.wait(5000);
+    // click 'reject'
+    cy.get('[data-cy="reject-expense-btn"]').within(() => {
+      cy.get('button').click({ force: true });
+    });
+    // click 'approve'
+    cy.get('[data-cy="approve-expense-btn"]').within(() => {
+      cy.get('button').click({ force: true });
+    });
+  });
+
+  it('mark pending application approved', () => {
+    // apply to brussels together asbl host
+    cy.visit('/brusselstogetherasbl');
+    //cy.contains('We are fiscally hosting 2 Collectives');
+    cy.contains('[data-cy="host-apply-btn"]', 'Apply').click({ force: true });
+    cy.get('#email').type('testuser@opencollective.com');
+    cy.wait(500);
+    cy.getByDataCy('signin-btn').click();
```

Comment:
Would be nice to use `cy.signup` here to ensure we start from a fresh/different user